chromium_src/chrome/browser/notifications/notification_platform_bridge_mac.mm

```diff
@@ -62,6 +64,20 @@ - (void)timerFired:(NSTimer *)timer {
 
 @end
 
+#define IsAtLeastOS10_15 IsAtLeastOS10_15_ChromiumImpl
```

Comment:
Shouldn't it be replaced with the name of our brave function below (which we probably might need to add Brave into the function name below to avoid conflict) instead of being `IsAtLeastOS10_15_ChromiumImpl`?
Otherwise the usage of `IsAtLeastOS10_15` in the original .mm will be replaced to try to call `
IsAtLeastOS10_15_ChromiumImpl` which would be undefined.

Reply:
Also, we should move this #define and #undef to the bottom of this file which we already have a line there to include this .mm file instead of adding a new one.

Reply:
> Shouldn't it be replaced with the name of our brave function below (which we probably might need to add Brave into the function name below to avoid conflict) instead of being `IsAtLeastOS10_15_ChromiumImpl`?
> Otherwise the usage of `IsAtLeastOS10_15` in the original .mm will be replaced to try to call ` IsAtLeastOS10_15_ChromiumImpl` which would be undefined.

Not according to the example I followed, but may I followed the wrong example. @bridiver Could you please check the code changes please